webrender/src/resource_cache.rs

```diff
@@ -186,6 +186,7 @@ pub struct ResourceCache {
     cached_images: ResourceClassCache<ImageRequest, CachedImageInfo>,
 
     // TODO(pcwalton): Figure out the lifecycle of these.
+    #[cfg_attr(not(feature = "webgl"), allow(dead_code))]
```

Comment:
why do we need to allow all these `dead_code` pieces instead of completely removing them?